src/components/atoms/Button/style.module.css

```diff
@@ -0,0 +1,15 @@
+.myButton{
```

Comment:
add an space after the selector name: ` .myButton {}` and also give a descriptive name to the selector